controllers/packagevariants/pkg/controllers/packagevariant/packagevariant_controller.go

```diff
@@ -56,8 +57,24 @@ type PackageVariantReconciler struct {
 const (
 	workspaceNamePrefix = "packagevariant-"
 
-	ConditionTypeStalled = "Stalled" // whether or not the packagevariant object is making progress or not
-	ConditionTypeReady   = "Ready"   // whether or not the reconciliation succeeded
+	ConditionTypeStalled        = "Stalled"                // whether or not the packagevariant object is making progress or not
+	ConditionTypeReady          = "Ready"                  // whether or not the reconciliation succeeded
+	ConditionTypePipelinePassed = "MutationPipelinePassed" // whether or not the mutation pipeline has completed successufully
```

Comment:
I suggest to rename it to `PackagePipelinePassed` or similar, since the success of the validation pipeline should be checked, as well, not just the mutation pipeline.

Reply:
This of course also applies to all other places where it is referenced as a "mutation pipeline" (e.g. in conditions)

Reply:
Done